### ISSUE TITLE
Failing test only on linux with node v0.10.30

### ISSUE THREAD

**Issue by springmeyer** [mapbox/mapbox-studio-classic#1433]
We currently have 3 jobs passing in the vs2015 branch (upcoming release) and only one failing: https://travis-ci.org/mapbox/mapbox-studio

That failing job has one failing test:

```
not ok 248 saves thumb
  ---
    operator: ok
    expected: true
    actual:   false
    at: null._onTimeout (/home/travis/build/mapbox/mapbox-studio/test/source.test.js:346:15)
  ...
```

Both @BergWerkGIS and I wonder if this is the same node v0.10.x bug seen elsewhere (api-maps).

**Comment by springmeyer** [comment]
upgrading to node v0.10.40 solves this failing test: https://travis-ci.org/mapbox/mapbox-studio/builds/76305474

**Comment by springmeyer** [comment]
Thanks @jakepruitt! This issue is now resolved by df18660343676ceff8e and 856103c

**Comment by jakepruitt** [comment]
Awesome, thanks for doing the update in mapbox-studio!
